Apps/NeuroscanAcquireLSL/nsatransporter.cpp

```diff
@@ -0,0 +1,113 @@
+#include "nsatransporter.h"
+
+NsaTransporter::NsaTransporter() {}
+
+void NsaTransporter::init() {
+    npackets = 0;
+    socket = new QTcpSocket;
+    connect(socket, SIGNAL(readyRead()), this, SLOT(data_handler()));
```

Comment:
The new connect signatur is more type safe, see https://wiki.qt.io/New_Signal_Slot_Syntax